### PR TITLE
Portfolio Accessibility

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
In this exercise, we have to check in PP the issues below if they need any changes.

 - **Page titles** - removed the My Portfolio and changed it to my name
- **Image text alternatives** - Changed - sent me an email
- **Text headings** - changed
- **Color contrast** - no issue found
- **Resize** - no issue found
- **Interaction** - no issue found
- **Moving content** - no issue found
- **Multimedia** - N/A
- **The basic structure of the page** - no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio</title>
+  <title>Nizamuddin</title>
```
Comment (omar-labana) [inline index.html:16]: - [ ] Kindly update your title to it adequately and briefly describes the content of the page, example:
`Nizamuddin's portifolio`
`Nizamuddin full-stack web developer`

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi Collaborators, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] Your portfolio still has some contrast ratio flaws. Kindly, check [this](https://www.aditus.io/button-contrast-checker/github-com-2021-11-09-at-22-26-00-312) audit and make sure that the `Contrast Checker` is catching no flaws.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 2]
```diff
@@ -194,93 +194,41 @@
...
   font-size: 14px;
-  font-family: inter, serif;
+  font-family: 'Inter', serif;
   margin-right: 10px;
   margin-top: 10px;
 }
 
 .see-prjct-btn button {
-  background-color: #ff6b00;
+  background: #f55800;
```
Comment (gabrielyea) [inline css/styles.css:203]: [OPTIONAL] I am sorry the previous review was not clear enough and didn't point where the problem was! :pray:   Great work trying to fix the contrast! Sadly the color change was not enough!. :disappointed: 

- [ ] Would you kindly fix the colors so they pass all the accessibility tests!
The contrast should have a ratio of 4.5:1, and as you can see below it doesn't meet the test requirements. :arrow_down: 
image

You can quickly test contrast values by using [contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/contrast())
## Check it out!
contrastChange

File: css/styles.css [hunk 3]
```diff
@@ -194,93 +194,41 @@
...
   margin-top: 10px;
+  font-weight: 700;
 }
 
 .see-prjct-btn button:hover {
   background-color: #fff;
-  border: 1px solid #ff6b00;
-  color: #ff6b00;
+  border: 1px solid #f55800;
+  color: #f55800;
```
Comment (gabrielyea) [inline css/styles.css:219]: [OPTIONAL] Same issue as above :arrow_up:  . I can see you tried to fix the contrast but it wasn't enough!

- [ ] I suggest you increase the contrast by either changing the colors or by using [contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/contrast())

**Review by gabrielyea — APPROVED** [review]
Hi @Nizamuddin4493  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and keep up the great work!

I left some `OPTIONAL` suggestions for you to try out!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**